dashel/dashel-posix.cpp

```diff
@@ -554,6 +554,24 @@ namespace Dashel
 		}
 	};
 	
+	//! Poll socket, used for polling only. A target "poll:sock=N" will call Hub::incomingData(stream)
+	//! exactly once when the socket N is polled with POLLIN in Hub::step.
```

Comment:
I would also explicitly state that this stream never actually reads or write data, as it aims to be used as an adapter for program reading/writing data by themselves.